client/a8c-for-agencies/sections/sites/sites-dashboard/style.scss

```diff
@@ -252,6 +257,8 @@
 				list-style: none;
 				margin: 0;
 				padding: 0;
+				overflow-y: auto;
+				height: calc(100vh - 165px);
```

Comment:
It's because of the header height. It calculates the parent height (the viewport) and subtracts the header size to determine the height of the scrollable view.